package.json

```diff
@@ -7,7 +7,7 @@
     "build": "bsb -make-world",
     "clean": "bsb -clean-world",
     "start": "bsb -make-world -w",
-    "test": "jest",
+    "test": "jest --bail --ci",
```

Comment:
Any objection to leaving "test" as is and adding "test:ci" with these options? Not sure how much it matters if we have bail. Tests would complete sooner but wouldn't necessarily know all the tests that are failing. If I run test during development I want know to know all the failures.

Reply:
Nope. Fixed in 629cb93